downloader/src/funTest/kotlin/MercurialTest.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.downloader.vcs
+
+import com.here.ort.downloader.Expensive
+
+import io.kotlintest.TestCaseContext
+import io.kotlintest.matchers.beGreaterThan
+import io.kotlintest.matchers.should
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.specs.StringSpec
+
+import java.io.File
+
+private const val REPO_URL = "https://bitbucket.org/creaceed/mercurial-xcode-plugin"
+
+class MercurialTest : StringSpec() {
+    private lateinit var outputDir: File
+
+    override val oneInstancePerTest: Boolean
```

Comment:
One (hopefully) last question: Why do we need this?

Reply:
Ok, figured it out and added an according code comment:

    // Required to make lateinit on outputDir work.